tests/test_entropy.py

```diff
@@ -0,0 +1,26 @@
+"""
+Testing entropy.py functionality
+"""
+
+import pathlib
+
+from test_git_parser import get_most_recent_commits
+
+from almanack.entropy import calculate_shannon_entropy
+
+
+def test_calculate_shannon_entropy(repository_paths: dict[str, pathlib.Path]) -> None:
+    """
+    Test calculate_shannon_entropy function.
+    """
+    file_sets = {
+        "high_entropy": ["high_entropy2.md", "high_entropy.md"],
+        "low_entropy": ["low_entropy.md"],
+    }
+    for label, repo_path in repository_paths.items():
+        source_commit, target_commit = get_most_recent_commits(repo_path)
+        entropies = calculate_shannon_entropy(
+            repo_path, source_commit, target_commit, file_sets[label]
+        )
+        for _, entropy in entropies.items():
```

Comment:
Consider adding a comparison for high entropy vs low entropy with an additional assert below (should one be higher than the other)?

Reply:
Referenced in test dataset issue